packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts

```diff
@@ -495,6 +495,8 @@ export const messages = {
   conflict_detect_diff_command_title: 'ファイルを比較',
 
   force_source_diff_text: 'SFDX: 組織のファイルとの差分を表示',
+  force_source_diff_remote_not_found:
+    'Selected file or directory was not found in org',
```

Comment:
Can you include Sonal and get her approval for this. Rest looks good.

Reply:
looks good to me.

Reply:
change to 'Selected file or directory was not found in the org'